NTupleMaker/bin/SynchNTupleProducer_2017.cpp

```diff
@@ -1451,8 +1505,11 @@ int main(int argc, char * argv[]){
   std::cout << "Total number of selected events = " << selEvents << std::endl;
   std::cout << std::endl;
   
+  cout<<"nonpionphotonctr ="<<nonpionphotonctr<<endl;
+
   file->cd("");
   file->Write();
+ConstitsPDG->Write();
 
```

Comment:
I'm not sure why the instruction `ConstitsPDG->Write();` is repeated two times (here and on L1492), but it should be irrelevant.
I would swap the instruction where you write the file and the one where you write the histogram in the file though, as I'm not sure whether the file will be updated or not.

Reply:
Removed the first instance of ConstitsPDG->Write();